pkg/drivers/kic/types.go

```diff
@@ -23,11 +23,13 @@ import (
 )
 
 const (
-	// Version is the current version of kic
+	// Version is the current version of kic -- also used as image tag
 	Version = "v0.0.36-1673540226-15630"
 
-	// SHA of the kic base image
-	baseImageSHA = "03c9592728381094cbd0ff9603f75ae6b485dd7a390c3e35f02ae5ec10f2f3ad"
+	// SHA of compressed image, used as a registry reference -- more reliable than a tag
+	distributionDigest = "03c9592728381094cbd0ff9603f75ae6b485dd7a390c3e35f02ae5ec10f2f3ad"
+	// SHA of SHAS of individual image layers -- content addressable
+	ContentDigest = "243d3449b30fd2029b685cafa1191f13fbce109441e8c74001ff370d444b1927"
```

Comment:
You would need one such Id per architecture, the one shown here is for a single architecture.

Reply:
removed ContentDigest on the last commit..
as commit message points out, it is an unnecessary information here.